src/zmq/zmqnotificationinterface.cpp

```diff
@@ -35,12 +35,14 @@ CZMQNotificationInterface* CZMQNotificationInterface::Create()
     std::map<std::string, CZMQNotifierFactory> factories;
     std::list<CZMQAbstractNotifier*> notifiers;
 
-    factories["pubhashblock"] = CZMQAbstractNotifier::Create<CZMQPublishHashBlockNotifier>;
-    factories["pubhashtx"] = CZMQAbstractNotifier::Create<CZMQPublishHashTransactionNotifier>;
+    factories["pubhashblock"]  = CZMQAbstractNotifier::Create<CZMQPublishHashBlockNotifier>;
```

Comment:
Can you revert these in this PR?

Do you know what is the clang-format recommendations for this type of alignment? (I am not sure myself.)

Reply:
If you want me to revert that formatting sure I can, I do not actually know what clang says on it. Couldn't find anything on it. I just know the old way it was looked bad and was annoying to read. Was hoping you could advise.

Reply:
I would prefer to remove formatting in this particular PR just to stick to keeping functional and formatting changes separate, for reasons described above. We do _try_ to stick to the clang-format convention, but I'm not sure exactly what that is for this situation.

There's a `clang-format` program which reads the rules in the [src/.clang-format](https://github.com/dashpay/dash/blob/e24b8ae50e416a4f79e5d4662bd57b35026d5e08/src/.clang-format) file and shows the correct format (or maybe it changes the code to align w/it) based on those rules.

Reply:
done